extern crate core;

use core::utils::error::Result;

fn main() -> Result<()> {
    core::start()
}
